lib/cli/parser.js

```diff
@@ -24,31 +36,34 @@ function getParser (debug = false) {
   // add the 'driver' and 'plugin' subcommands
   addExtensionsToParser(sharedArgs, subParsers, debug);
 
-  // modify the parseArgs function to insert the 'server' subcommand if the
+  // modify the parse_args function to insert the 'server' subcommand if the
   // user hasn't specified a subcommand or the global help command
-  parser._parseArgs = parser.parseArgs;
-  parser.parseArgs = function (args, namespace) {
+  parser._parse_args = parser.parse_args;
```

Comment:
shouldn't we use bind?

Reply:
yeah, probably a good idea, even though it seemed to work fine without for forever. will update.